code/modules/mob/living/carbon/human/species.dm

```diff
@@ -602,6 +602,12 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			blush_overlay.color = COLOR_BLUSH_PINK
 			standing += blush_overlay
 
+		// tears
+		if (HAS_TRAIT(species_human, TRAIT_CRYING))
+			var/mutable_appearance/tears_overlay = mutable_appearance('icons/mob/human_face.dmi', "tears", -BODY_ADJ_LAYER)
+			tears_overlay.color = COLOR_DARK_CYAN
+			standing += tears_overlay
```

Comment:
this should all belong to the head or eyes, it shouldnt be an arbitrary species thing, im trying to get rid of this awful proc.

Reply:
The crying overlay has been moved to the eyes. I've also added a proc similar to what the eyes have to the head and moved the blush emote to there.